src/main/java/uk/gov/cshr/civilservant/controller/CivilServantController.java

```diff
@@ -61,12 +82,44 @@ public CivilServantController(CivilServantRepository civilServantRepository,
         return getResourceResponseEntity(optionalCivilServant);
     }
 
+    @PatchMapping("/manager")
+    @PreAuthorize("isAuthenticated()")
+    public ResponseEntity<Resource<CivilServantResource>> check(@RequestParam(value = "email") String email) throws NotificationClientException {
+        IdentityFromService lineManager = new IdentityFromService();
```

Comment:
NAMING - Service and variable names are inconsistent. I have no idea without delving into code that what the class achieves.

Reply:
A bit like get() , and list() perhaps ...

Reply:
Semantics. it checks something about a civil servant. 
Meaning is a subjective ideology. I would say this to be as meaningful as get() or list() and certainly no less than "process" or "getResourceResponseEntity"